spec/i18n_routable/i18n_routable_spec.rb

```diff
@@ -149,55 +153,55 @@ def resolve(path, http_verb=:get)
 
     it 'should understand the difference between display locales and backend locales' do
       I18n.locale = :gibb
-      new_post_url.should eql "http://www.example.com/gibberish/posts/the-new"
+      new_post_url.should eql "http://www.example.com/gibb/posts/the-new"
```

Comment:
For gibberish specifically, it is fine if we just make it gibb, since that is the locale we use for internal testing.

But I worry that we use that functionality (hash of display_locale => rails_locale) in our usage of i18n_routable, and this test was ensuring it worked (as a side-effect).

@mdimas Do you know if we still do that?